ADAL/ADALAutomation/ADAutoResultViewController.m

```diff
@@ -1,116 +1,53 @@
-
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
 
 #import "ADAutoResultViewController.h"
-#import "ADAutoTextAndButtonView.h"
 
 @interface ADAutoResultViewController ()
-{
-    ADAutoTextAndButtonView* _myView;
-}
+
+@property (strong, nonatomic) IBOutlet UITextView *resultInfo;
+@property (strong, nonatomic) IBOutlet UITextView *resultLogs;
 
 @end
 
 @implementation ADAutoResultViewController
-{
-    NSString* _result;
-}
-
-- (id)initWithResultJson:(NSString*) result
-{
-    if (!(self = [super init]))
-    {
-        return nil;
-    }
-    
-    _result = result;
-    
-    return self;
-}
-
-- (void)loadView
-{
-    UIView* contentView = [[UIView alloc] initWithFrame:UIScreen.mainScreen.bounds];
-    contentView.backgroundColor = UIColor.whiteColor;
-    self.view = contentView;
-    _myView = [[ADAutoTextAndButtonView alloc] initWithFrame:UIScreen.mainScreen.bounds];
-    [contentView addSubview:_myView];
-    
-    _myView.textView.text = _result;
-    _myView.textView.accessibilityIdentifier = @"resultInfo";
-    
-    [_myView.actionButton setTitle:@"Done" forState:UIControlStateNormal];
-    [_myView.actionButton addTarget:self
-                             action:@selector(done:)
-                   forControlEvents:UIControlEventTouchUpInside];
-    _myView.actionButton.accessibilityIdentifier = @"resultDone";
-    
-    NSDictionary* views = @{ @"textAndButtonView" : _myView,
-                             @"topLayoutGuide" : self.topLayoutGuide,
-                             @"bottomLayoutGuide" : self.bottomLayoutGuide };
-    
-    NSArray* verticalConstraints =
-    [NSLayoutConstraint constraintsWithVisualFormat:@"V:[topLayoutGuide]-[textAndButtonView]-[bottomLayoutGuide]"
-                                            options:0
-                                            metrics:nil
-                                              views:views];
-    
-    NSArray* horizontalConstraints =
-    [NSLayoutConstraint constraintsWithVisualFormat:@"H:|-[textAndButtonView]-|"
-                                            options:0
-                                            metrics:nil
-                                              views:views];
-    
-    [self.view addConstraints:verticalConstraints];
-    [self.view addConstraints:horizontalConstraints];
-}
 
 - (void)viewDidLoad {
     [super viewDidLoad];
     // Do any additional setup after loading the view.
+    
+    self.resultInfo.text = self.resultInfoString;
+    self.resultLogs.text = self.resultLogsString;
```

Comment:
@unpluggedk, shouldn't the logs element be named `adalLogs`?
It should be consistent with the other automation test apps ([Android](https://github.com/AzureAD/azure-activedirectory-library-for-android/blob/weij/automationTestApp/automationtestapp/src/main/res/layout/activity_result.xml#L51), [Desktop](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/blob/dev/automation/WinFormsAutomationApp/MainForm.Designer.cs#L45))